CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v2.0.3, 23 June 2021
```

Comment:
Shouldn't this be a minor release rather than patch?

Reply:
Yes just realised this myself 🙈 

Too many versions 🤯